### PR TITLE
[CCXT] add reduce only to create order (future exchange tentacles must implement it)

### PR DESCRIPTION
**Pull request by techfreaque**
requires PR in tentacles -> [CreateOrder] add reduce only parameter

see binance usdm branch as an example

### CODE REVIEW

**Comment by GuillaumeDSM** [comment]
ok for me once https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/804 is rebased

**Comment by techfreaque** [comment]
> ok for me once [Drakkar-Software/OctoBot-Tentacles#804](https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/804) is rebased

Done, + I added order tags

**Comment by GuillaumeDSM** [comment]
we can merge it when the CI will pass

**Comment by techfreaque** [comment]
> we can merge it when the CI will pass

Tests are passing now

**Comment by techfreaque** [comment]
I removed the order tags from this PR as well

File: octobot_trading/exchanges/types/rest_exchange.py [hunk 1]
```diff
@@ -242,33 +246,34 @@ def _ensure_order_details_completeness(self, order, order_required_fields=None,
 
     async def _create_specific_order(self, order_type, symbol, quantity: decimal.Decimal, price: decimal.Decimal = None,
                                      side: enums.TradeOrderSide = None, current_price: decimal.Decimal = None, 
-                                     stop_price: decimal.Decimal = None, params=None) -> dict:
+                                     stop_price: decimal.Decimal = None, reduce_only: bool = False, params=None) -> dict:
         created_order = None
         float_quantity = float(quantity)
         float_price = float(price)
+        float_stop_price = float(stop_price)
```
Comment (Herklos) [inline octobot_trading/exchanges/types/rest_exchange.py:253]: :+1:

**Review by Herklos — APPROVED** [review]

**Pull request merged**